main.py

```diff
@@ -192,23 +194,24 @@ async def add_remove_reliable_role(self):
             guild_id: int = self.reliable_role.guild.id
 
             if len(users) == 1:
-                sql_stmt: str = (f'SELECT member_id, karma FROM {Bot.TABLE_MEMBERS} '
+                sql_stmt: str = (f'SELECT member_id, correct, wrong, karma FROM {Bot.TABLE_MEMBERS} '
                                  f'WHERE member_id = {tuple(users)[0]} AND server_id = {guild_id}')
             else:
-                sql_stmt: str = (f'SELECT member_id, karma FROM {Bot.TABLE_MEMBERS} '
+                sql_stmt: str = (f'SELECT member_id, correct, wrong, karma FROM {Bot.TABLE_MEMBERS} '
                                  f'WHERE server_id = {guild_id} AND member_id IN {tuple(users)}')
 
             cursor.execute(sql_stmt)
-            result: Optional[list[tuple[int, float]]] = cursor.fetchall()
+            result: Optional[list[tuple[int, int, int, float]]] = cursor.fetchall()
             conn.close()
 
             if result:
                 for data in result:
-                    member_id, karma = data
+                    member_id, correct, wrong, karma = data
                     if karma != 0:
                         member: Optional[discord.Member] = self.reliable_role.guild.get_member(member_id)
                         if member:
-                            if karma > RELIABLE_ROLE_THRESHOLD:
+                            accuracy: float = correct / (correct + wrong)
+                            if karma >= RELIABLE_ROLE_KARMA_THRESHOLD and accuracy >= RELIABLE_ROLE_ACCURACY_THRESHOLD:
```

Comment:
Round off the accuracy and the karma. Earlier we rounded off accuracy to two places of decimal (if accuracy was a percentage) or to four places of decimal (if it was a fraction). Something like that has to be implemented.